exec/java-exec/src/main/java/org/apache/drill/exec/store/sys/VersionIterator.java

```diff
@@ -73,5 +78,33 @@ public void remove() {
     throw new UnsupportedOperationException();
   }
 
+  /**
+   *
+   * @return version number x.y.z from Manifest file
+   */
+  private static String getDrillVersionFromManifest() {
+    String appName = "";
+    String appVersion = "";
+    try {
+      Enumeration<URL> resources = VersionInfo.class.getClassLoader()
+              .getResources("META-INF/MANIFEST.MF");
+      while (resources.hasMoreElements()) {
+        Manifest manifest = new Manifest(resources.nextElement().openStream());
+        // check that this is your manifest and do what you need or
+        // get the next one
+        appName = manifest.getMainAttributes()
+                .getValue("Implementation-Title");
+        if (appName != null && appName.toLowerCase().contains("drill")) {
+          appVersion = manifest.getMainAttributes()
+                  .getValue("Implementation-Version");
+        }
+      }
+    } catch (IOException except) {
+      appVersion = "";
```

Comment:
Do you want me to move the getVersion method in this package:
https://github.com/apache/drill/tree/master/common/src/main/java/org/apache/drill/common/util ?

Should I move the full VersionIterator there? and call it from the store.sys?